ui_testing/pages/order_page.py

```diff
@@ -104,6 +105,37 @@ def create_new_order(self, material_type='', article='', contact='', test_plan='
         elif test_unit:
             self.set_test_unit(test_unit=test_unit)
         self.save(save_btn='order:save')
+        
+    def set_departments(self, departments=''):
+        if departments:
+            self.base_selenium.select_item_from_drop_down(element='order:departments', item_text=departments)
+        else:
+            self.base_selenium.select_item_from_drop_down(element='order:departments')
+            return self.get_departments()
+
+    def get_departments(self):
```

Comment:
u changes the code of this method, Did u check the other test cases which r using the old code of it? You need to make sure that these changes dont break other test cases